### DISCUSSION TITLE
FileSearch usage

### DISCUSSION THREAD

**Discussion by Bodekaer** [RageAgainstThePixel/com.openai.unity#248] [Q&A]
@StephenHodgson I am testing out the FileSearch/Vector Stores update, and I realized there is no documentation on best practice for activating the FileSearch on an Assistant along with other local tools and how to attach a vector store by an existing vector store id.
Could you share a brief code example?

**Comment by Bodekaer** [comment]
Update: I figured it out now using `ToolResources` in `CreateAssistantRequest`. 
A quick example in the documentation would likely still be helpful for others :)

**Reply by StephenHodgson** [reply]
Yeah I think I also need to update the file search to include optional parameters. 

I was going to do that in a future update.

**Reply by sirperalta** [reply]
> Update: I figured it out now using `ToolResources` in `CreateAssistantRequest`. A quick example in the documentation would likely still be helpful for others :)

I'm struggling with that. Please, can you show me how did you did it?
